pom.xml

```diff
@@ -1496,6 +1495,7 @@
               <configuration>
                 <compilerArgs>
                   <arg>${compiler.default.pkginfo.flag}</arg>
+                  <arg>-parameters</arg>
```

Comment:
Moving this here works for the goal, just curious about something, when I look at the compiler args by doing mvn -X compile it shows:
```
[DEBUG]   (f) compilerArgs = [-Xpkginfo:always, -parameters]
```
so it seems that it is overwriting the parameters in configuration (the -Xlint ones), isn't this an error? Maybe a different one?

Reply:
You are right, but IMHO that's another issue ;) I will propose another PR to merge (with `combine.children=append` configuration to the `maven-compiler-plugin`) as the `-Xlint*` options are not actually used at all (for a long time, maybe since the beginning of the project ;)).

Reply:
Interesting point is that enabling these options breaks the build because of -Werror, can you please create the JIRA for the merge/append compiler args I took a look and gradle tooks well the options but has but does not complain, probably -Werror is missing there.